apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/page.tsx

```diff
@@ -60,101 +72,89 @@ export default async function Pool({
-                        <span className="text-md pl-2 text-black">
-                          {" "}
-                          {tokenGarden?.symbol}
-                        </span>
-                      </div>
-                    )}
-                    <div className="text-md stat-title">
-                      Points System:{" "}
-                      <span className="text-md pl-2 text-black">
-                        {pointSystems[pointSystem]}
-                      </span>
-                    </div>
-                  </div>
-                </div>
-                <div className="flex flex-1 flex-col items-center space-y-2">
-                  <p className="text-md stat-title text-xl font-semibold">
-                    Proposals type accepted:
-                  </p>
-                  <div className="flex w-full items-center justify-evenly">
-                    <Badge type={proposalType} />
-                  </div>
-                </div>
-              </div>
-              <div className="flex items-center justify-center"></div>
-            </div>
-            <div className=" flex">
-              {[...Array(6)].map((_, i) => (
-                <Image
-                  key={i}
-                  src={gardenLand}
-                  alt="garden land"
-                  className=""
-                />
-              ))}
-            </div>
-          </section>
-          {/* Pool metrics: for now we have funds available and spending limit */}
-          {isEnabled && (
-            <>
-              {poolTypes[proposalType] !== "signaling" && (
-                <PoolMetrics
-                  alloInfo={alloInfo}
-                  poolId={poolId}
-                  balance={poolAmount}
-                  strategyAddress={strategyAddr}
-                  strategy={strategyObj}
-                  communityAddress={communityAddress}
-                  tokenGarden={tokenGarden}
-                  pointSystem={pointSystem}
-                  chainId={parseInt(chain)}
-                  spendingLimitPct={spendingLimitPct}
-                />
-              )}
-              {/* Proposals section */}
-              <Proposals
-                strategy={strategyObj}
-                alloInfo={alloInfo}
-                communityAddress={communityAddress}
-                createProposalUrl={`/gardens/${chain}/${garden}/pool/${poolId}/create-proposal`}
-                proposalType={proposalType}
+          {poolTypes[proposalType] == "funding" && (
+            <Statistic label="funding token" icon={<InformationCircleIcon />}>
+              <Badge
+                isCapitalize
+                label={tokenGarden.symbol}
+                icon={<Square3Stack3DIcon />}
               />
-            </>
+            </Statistic>
+          )}
+
+          <Statistic
+            label="voting weight system"
+            icon={<InformationCircleIcon />}
+          >
+            <div className="flex flex-col gap-3 sm:flex-row">
```

Comment:
Just making sure we both understand what is done here with the sm:
You wanted to above mobile resolution right?
SM = resolution >= 640px and here is how we target mobile devices:
![image](https://github.com/1Hive/gardens-v2/assets/13890618/63ee2e48-e782-4c61-96c4-1639f9811d5c)

Reply:
Source: https://tailwindcss.com/docs/responsive-design

Reply:
Yeah, so we apply classes for mobile and in the case we wanna have different styles above that resolution we do sm: ... , or md: ...
For the moment we just add some sm: ... styles, so we have at least components and page details in a nice order and stacked